config/betterquesting/DefaultQuests/Quests/MultipleQuestLine/ETOOMANYKEYS-AAAAAAAAAAAAAAAAAAAIYQ==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "ERROR: Too many keys to fit a 104 key keyboard.\n\nThere are lots of keybinds in this modpack, to avoid conflicts most of them are unbinded by default and only those available from the start of the game are binded.\nHere is a list of what some of them do :\n\n\nBetterQuesting\nGRAVE - Open Questbook\n\nBattlegear\nR - Draw Weapons - It\u0027s a second hotbar that holds weapons\n\nDraconic Evolution\nV - Place the item you are holding on the block you are looking at\n\nJourneymap\nB - Create a Waypoint\nJ - Open Fullscreen Map\n\u003d - Zoom In (works on minimap too)\n- - Zoom out\n\nInventory Tweaks\nR - Sort Inventory\n\nShare Where I am\nINSERT - Places a temporary beam where you are looking at that will be seen by you and other players\nSHIFT + INSERT - Places a temporary beam where you are that will be seen by you and other players\nCTRL + INSERT - Shares with other players the position you are looking at\nCTRL + SHIFT + INSERT - Shares with other players your position\n\nWaila - Shows informations of what you are looking at\nNUMPAD1 - show/hide the Waila HUD\nNUMPAD3 - shows how to craft what you are looking at\nNUMPAD4 - shows the crafting usages of the thing you are looking at\n"
+      "desc:8": "ERROR: Too many keys to fit a 104 key keyboard.\n\nThere are lots of keybinds in this modpack, to avoid conflicts most of them are unbinded by default and only those available from the start of the game are binded.\nHere is a list of what some of them do :\n\n\nBetterQuesting\nGRAVE - Open Questbook\n\nBattlegear\nR - Draw Weapons - It\u0027s a second hotbar that holds weapons\n\nDraconic Evolution\nV - Place the item you are holding on the block you are looking at\n\nJourneymap\nB - Create a Waypoint\nJ - Open Fullscreen Map\n\u003d - Zoom In (works on minimap too)\n- - Zoom out\n\nInventory Tweaks\nR - Sort Inventory\n\nShare Where I am\nINSERT - Places a temporary beam where you are looking at that will be seen by you and other players\nSHIFT + INSERT - Places a temporary beam where §oyou§r are that will be seen by you and other players\nCTRL + INSERT - Shares the position you are looking at with other players\nCTRL + SHIFT + INSERT - Shares your position with other players\n\nWaila - Shows information about what you are looking at\nNUMPAD1 - show/hide the Waila HUD\nNUMPAD3 - shows how to craft what you are looking at\nNUMPAD4 - shows the crafting usages of the thing you are looking at\n"
```

Comment:
looks like you reverted your last fix here. also explains the merge conflict

Reply:
Yes, I resolved this conflict on local, haven't pushed yet. Working on your suggestions.